### DISCUSSION TITLE
Is there a way to ask for Android authentication?

### DISCUSSION THREAD

**Discussion by eamonburns** [termux/termux-app#3691] [Q&A] [answered]
Is there a way (from inside a python script or something) to ask the Android API to authenticate the user? For instance, when I open my bank app, it requires me to scan my fingerprint (or login with a username/password), could I do something like that?

**Comment by agnostic-apollo** [comment, marked as answer]
`termux-fingerprint`

https://wiki.termux.com/wiki/Termux:API
